docs/carts/cart_interactions/index.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Cart interactions
+parent: Carts
+has_children: true
+nav_order: 3
+---
+
+# Cart interactions
+Interactions with a customer's cart are enabled by including [form tags]({% link docs/reference/tags/index.md %}) in block or template code.
+
+- [Add line items to the cart]({% link docs/carts/cart_interactions/add_line_items.md %})
+- [Remove line items from the cart]({% link docs/carts/cart_interactions/remove_line_items.md %})
+- [Clear the cart]({% link docs/carts/cart_interactions/clear_cart.md %})
+
+
+## Additional form tag parameters
```

Comment:
I'd leave this just in the clear_cart guide, rather than having a copy here as well.